OscLib/OscCalcAnalytic.cxx

```diff
@@ -344,7 +344,7 @@ namespace osc::analytic
     default: abort();
     }
   }
-
+*/
```

Comment:
since this function is now pulled in from `Cache.h`, this function should be deleted instead of just commented out